src/utils/management/commands/send_email.py

```diff
@@ -0,0 +1,273 @@
+            logger.error(
+                self.style.ERROR(
+                    f'TO and CC must have fewer than {batch_size} addresses'
+                )
+            )
+            return
+
+        if not options['bcc']:
+            bcc = []
+        else:
+            bcc_file = os.path.join(settings.BASE_DIR, options['bcc'])
+            try:
+                with open(bcc_file, 'r') as ref:
+                    bcc = json.loads(ref.read())
+            except FileNotFoundError:
+                logger.error(
+                    self.style.ERROR(f'File not found at {bcc_file}')
+                )
+                if verbosity >= 1:
+                    self.print_help('manage.py', 'send_email')
+                return
+
+        if len(bcc) > batch_size:
+
+            batch_num = 1
+            log_dict['action_text'] = f'{log_subject} (batch {batch_num})'
+
+            custom_reply_to_setting = setting_handler.get_setting(
+                'general',
+                'replyto_address',
+                journal,
+            )
+            if custom_reply_to_setting and custom_reply_to_setting.value:
+                reply_to = (custom_reply_to_setting.value,)
+            else:
+                logger.error(
+                    self.style.ERROR(
+                        f'replyto_address setting needed for BCC list '
+                        f'length over {batch_size}'
+                    )
+                )
+                return
+
+        if options['confirm']:
+            logger.info(
+                f'\n\n'
+                f'Preparing to send email from {request.site_type}\n\n'
+                f'Subject: {subject}\n\n'
+                f'TO: {to}\n\n'
+                f'CC: {cc}\n\n'
+                f'BCC: {bcc}\n\n'
+                f'Body: \n{body}\n\n'
+            )
+
+            send = input('Send? (yes/no): ')
+
+            if not send.lower() == 'yes':
+                logger.info('Email discarded')
+                return
+
+        # Send batch 1
+        bcc_chunk = bcc[:batch_size]
+        send_email_with_body_from_user(
+            request,
+            subject,
+            to,
+            body,
+            log_dict=log_dict,
+            cc=cc,
+            bcc=bcc_chunk,
+        )
+
+        # Send the rest with noreply address in TO and empty CC
+        if len(bcc) > batch_size:
+            to = reply_to
+            cc = []
+            for i, bcc_chunk in enumerate(
+                chunked(bcc[batch_size:], batch_size)
+            ):
+                batch_num = i + 2
+                log_dict['action_text'] = f'{log_subject} (batch {batch_num})'
+                send_email_with_body_from_user(
+                    request,
+                    subject,
+                    to,
+                    body,
+                    log_dict=log_dict,
+                    cc=cc,
+                    bcc=bcc_chunk,
+                )
+
+        logger.info(self.style.SUCCESS('Email sent'))
```

Comment:
can we add a short sleep timer (a fraction of a second) here in between batches? It is  particularly important when using email backends like Mailgun, to make sure we don't overrun limits